charts/k8ssandra-operator/values.yaml

```diff
@@ -33,7 +33,7 @@ image:
   pullPolicy: IfNotPresent
 
   # -- Tag of the k8ssandra-operator image to pull from image.repository
-  tag: 99c58003
+  tag: 5b95fc98
```

Comment:
Should we go ahead and bump to the tag that was published this morning after the medusa integration got merged? `b05f3ec8`

I'll try to test with that image today.

Reply:
Ah, never mind, there's a new CRD for the backup type, so we can't use it without more changes.  We can just follow this up with that.